app/views/flower_shops/edit.html.erb

```diff
@@ -0,0 +1,11 @@
+<h1><%= @flower_shop.name %></h1>
+
+<%= bootstrap_form_with url: "/flower_shops/#{@flower_shop.id}", method: :patch, local: true do |form| %>
+  <%= form.text_field :name, value: @flower_shop.name %><br/>
+  <%= form.form_group :same_day_delivery, label: { text: "Same Day Delivery?" } do %><br/>
+    <%= form.radio_button :same_day_delivery, true, checked: @flower_shop.same_day_delivery, label: "Yes", inline:true %>
+    <%= form.radio_button :same_day_delivery, false, checked: @flower_shop.same_day_delivery, label: "No", inline:true %>
```

Comment:
You'll want the `false` option to be checked (read: true) when same_day_delivery is false, so you're missing a `!` like so:

`checked: !@flower_shop.same_day_delivery`